packages/terra-divider/docs/README.md

```diff
@@ -0,0 +1,23 @@
+# Terra Divider
+
+The divider component is used to visually separate content on a layout.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-divider`
+  - `yarn add terra-divider`
```

Comment:
Do we even use yarn? Thinking this can be removed.

Reply:
These instructions are for our consumers, not necessarily us as library authors. Some of our consumers use yarn to install our packages so we provide doc on how to install with both npm and yarn.